rules_building_block/persistence_suspicious_file_opened_through_editor.toml

```diff
@@ -0,0 +1,106 @@
+[metadata]
+creation_date = "2023/07/25"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/25"
+integration = ["endpoint"]
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for the potential edit of a suspicious file. In Linux, when editing a file through an editor, a 
+temporary .swp file is created. By monitoring for the creation of this .swp file, we can detect potential file edits of
+suspicious files. The execution of this rule is not a clear sign of the file being edited, as just opening the file 
+through an editor will trigger this event. Attackers may alter any of the files added in this rule to establish
+persistence, escalate privileges or perform reconnaisance on the system. 
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*", "endgame-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Suspicious File Edit"
+risk_score = 21
+rule_id = "3728c08d-9b70-456b-b6b8-007c7d246128"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Persistence", "Tactic: Privilege Escalation", "Data Source: Elastic Endgame", "Rule Type: BBR"]
+timestamp_override = "event.ingested"
+type = "eql"
+building_block_type = "default"
+
+query = '''
+file where event.action in ("creation", "file_create_event") and file.extension == "swp" and 
```

Comment:
Do Linux file creation events cover modifications?

Reply:
Yes, like it states in the description, editing a file through an editor creates a temporary .swp file, which indicates a file edit event.